sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/Microsoft.Azure.WebJobs.Extensions.SignalRService.csproj

```diff
@@ -11,24 +11,24 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <!--Pin the versions of packages that may be upgraded during the process of Azure SDK moving all dependencies to the 8.x line to avoid incompatibility, until a compatible dependency gragh is figured out and finalized.-->
     <PackageReference Include="MessagePack" />
-    <PackageReference Include="Microsoft.Extensions.Azure" />
-    <PackageReference Include="Microsoft.Azure.Functions.Extensions" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" VersionOverride="3.0.37"/>
     <PackageReference Include="Microsoft.Azure.SignalR.Management" />
     <PackageReference Include="Microsoft.Azure.SignalR" />
     <PackageReference Include="Microsoft.Azure.SignalR.Protocols" />
     <PackageReference Include="Microsoft.Azure.SignalR.Serverless.Protocols" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" />
+    <PackageReference Include="Microsoft.Extensions.Azure" VersionOverride="1.10.0"/>
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" VersionOverride="6.35.0"/>
+  </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Microsoft.AspNetCore.Http.Connections" />
     <!--
         Adding as a local override, as packages should not generally take this dependency.  It is
-        needed here due to a transitive version conflict due to the Microsoft.Azure.Functions.Extensions
+        needed here due to a transitive version conflict due to the 2.x version of Microsoft.* packages.
         reference.
     -->
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" VersionOverride ="8.0.1" />
```

Comment:
This should no longer be necessary with the removal of Azure.Functions.Extensions

Reply:
As I mentioned in a previous [comment](https://github.com/Azure/azure-sdk-for-net/issues/48163#issuecomment-2655881652), the error is still there as long as we meet the following three conditions:
1. We have dependencies on `Microsoft.Extensions.DependencyInjection (v2.2.0)` and `Microsoft.Extensions.DependencyInjection.Abstractions (v8.0.2)`.
2. We have .NET Standard 2.0 target framework
3. We use `ServiceCollection` symbol. (Other functions extensions package happen to not use it)

And regarding to which packages introduce transitive dependencies  `Microsoft.Extensions.DependencyInjection (v2.2.0)`:
1. Microsoft.Azure.WebJobs
2. Microsoft.Azure.SignalR.*  . For these packages, we could manage to get rid of `Microsoft.Extensions.DependencyInjection (v2.2.0)` by ourselves.

Reply:
Microsoft.Azure.WebJobs does not introduce a transitive dependency conflict when using the repository central version.   Every functions extension package in this repository depends on it, and only WebPubSub has this conflict.

Reply:
For other users who rely on `Microsoft.Azure.WebJobs`, such as functions extensions users, if they introduce `ServiceCollection` symbol in their own code, they will also get that conflict. Therefore, I believe the resolution of this conflict issue from `Microsoft.Extensions.Hosting` side is not only meaningful to SignalR and Web PubSub users, but also has potential significance for a broader user base.